docs/operating-scylla/nodetool-commands/status.rst

```diff
@@ -2,6 +2,9 @@ Nodetool status
 ===============
 **status** - This command prints the cluster information for a single keyspace or all keyspaces.
 
+The keyspace argument is required to calculate effective ownership information (``Owns`` column).
+For tablet keyspaces, a table is also required for effective ownership.
```

Comment:
btw, where does this limitation stem from? 

We could sum up vnode and tablet ranges.

Reply:
> The example below demonstrates otherwise (it has no keyspace argument and has the Owns column filled).

I think the example is wrong, there is a mismatch between the invocation and the output.

Reply:
> btw, where does this limitation stem from?
> 
> We could sum up vnode and tablet ranges.

We have decided aginst doing this. Requesting ownership without a keyspace will return the following error from the server side:
Non-system keyspaces don't have the same replication settings, effective ownership information is meaningless